doc/tutorials/workshop/content/exercises/12-common-rules.md

```diff
@@ -0,0 +1,75 @@
+---
+Title: All types of rules
+PrevPage: 11-node-rules
+NextPage: ../finish
+---
+Common types of rules
+===================
+
+The rules we created in the previous chapter leveraged the `yamlfile_value`
+and the `file_permissions` template from CaC/content project. There are many
+more templates available in the project and each one enable us to quickly
+create rules with a specific behavior.
+
+In this section we will go through the most common checks when assessing the
+security posture of a cluster and the templates used to implement them.
+
+### Checking Kubernetes resources
+
+For example, to ensure that anonyous check for 
+
+The `./utils/add_platform_rule.py` script from last section provides a very convenient
+way to write rules checking Kubernetes configuration.
+
+Below you'll see more examples of the resources that can be checked.
+#### Checking for kubeconfig
+
+### Checking a new platform resource
+
+#### Granting the operator permssion to access the new resource (?)
+
+
+### Checking ownership and permissions of files
+
+A very common requirement is to ensure that files in the cluster node have
```

Comment:
This might also include the file permissions as they're mounted into containers, right?

Reply:
Hmm, but does the CO scan the files mounted in the containers?

Reply:
I should have rephrased my question. Could this be used to check permissions mounted into the container?

I only ask because there are some controls in CIS that are now assessing permissions inside the container, as opposed to just on the host.